creator-node/src/middlewares/premiumContent/premiumContentMiddleware.ts

```diff
@@ -1,13 +1,14 @@
+import type { CustomRequest } from '../../apiHelpers'
+
 import {
-  sendResponse,
+  sendResponseWithMetric,
   errorResponseServerError,
   errorResponseForbidden,
   errorResponseUnauthorized,
   errorResponseBadRequest
 } from '../../apiHelpers'
-import { NextFunction, Request, Response } from 'express'
+import { NextFunction, Response } from 'express'
```

Comment:
nit: `import type { NextFunction, Response } from 'express'`

Reply:
can you import type here if `NextFunction` and `Response` are interfaces?

Reply:
got ts error trying to import this, so gonna resolve